src/all/eternalmangas/src/eu/kanade/tachiyomi/extension/all/eternalmangas/EternalMangas.kt

```diff
@@ -57,23 +58,33 @@ open class EternalMangas(
         query: String,
         filters: FilterList,
     ): MangasPage {
-        val body = response.body.string()
         val data = client.newCall(GET(dataUrl)).execute().body.string().split("\n")
```

Comment:
Use a DTO for this? Or [destructor](https://kotlinlang.org/docs/destructuring-declarations.html). it is difficult to tell from the code what `data[0]` is opposed to `data[3]` without seeing the actual response from the site.